practitioner-service/src/main/java/uk/ac/ox/ndph/mts/practitioner_service/service/EntityService.java

```diff
@@ -17,4 +19,6 @@
     String savePractitioner(Practitioner practitioner);
 
     String saveRoleAssignment(RoleAssignment roleAssignment);
+
+    List<RoleAssignment> getRoleAssignmentsByIdentifier(String practitionerId);
```

Comment:
perhaps say it is by practitioner identifier? Also the param name might be confusing since Id and Identifier aren't the same thing.